com.nextcloud.desktopclient.nextcloud.metainfo.xml

```diff
@@ -36,4 +36,5 @@ to the client, you can work with your files even when you are not online!</p>
     <release date="2019-07-22" version="2.5.3"/>
   </releases>
   <content_rating type="oars-1.1"/>
+  <translation type="gettext">https://www.transifex.com/nextcloud/nextcloud/</translation>
```

Comment:
the translation tag is not used for this. It's used to compute if your currently used language is available on that said app. If you want to link to where the user could contribute translations, you should use the url tag instead.

Reply:
According to [the XDG Appstream specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html), that's not the case, unless Flathub works differently:
> **translate**
>
> URLs of this type should point to a webpage where users can submit or modify translations of the upstream project.
>
> Typically this should be a link to the project page in Weblate, Transifex or Zanata, but could also be a link to an upstream-hosted wiki page describing how to send translations upstream.

Reply:
You're looking at the url of type translate, not the [translation](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-translation) tag here.

Reply:
I saw that, too. I'm not sure what the difference is, considering:
> `<translation/>` #
> 
> The `<translation/>` tag is an optional tag which can be added to specify the translation domain used for this software component. It may be used by the AppStream distro metadata generator to determine the translation status of the respective software (e.g. which languages the software is translated into and how complete the translations are).

I also used [this](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Localization.html#spec-l10n-example) as a reference for the `<languages />` and `<lang />` tags in [my other pull request](https://github.com/flathub/com.nextcloud.desktopclient.nextcloud/pull/37) (strictly speaking it's for a different metadata file, though the file still validates with `appstreamcli`).